geode-core/src/test/java/org/apache/geode/management/internal/configuration/ClusterConfigLocatorRestartDUnitTest.java

```diff
@@ -61,19 +57,15 @@ public void onDisconnect(InternalDistributedSystem sys) {
   public void serverRestartsAfterLocatorReconnects() throws Exception {
     IgnoredException.addIgnoredException("org.apache.geode.ForcedDisconnectException: for testing");
 
-    Properties props = new Properties();
-    props.setProperty(ConfigurationProperties.MAX_WAIT_TIME_RECONNECT, "5000");
-    MemberVM locator0 = rule.startLocatorVM(0, props);
+    MemberVM locator0 = rule.startLocatorVM(0);
 
-    rule.startServerVM(1, props, locator0.getPort());
-    MemberVM server2 = rule.startServerVM(2, props, locator0.getPort());
+    rule.startServerVM(1, locator0.getPort());
+    MemberVM server2 = rule.startServerVM(2, locator0.getPort());
 
     addDisconnectListener(locator0);
 
-    server2.invokeAsync(() -> MembershipManagerHelper
-        .crashDistributedSystem(InternalDistributedSystem.getConnectedInstance()));
-    locator0.invokeAsync(() -> MembershipManagerHelper
-        .crashDistributedSystem(InternalDistributedSystem.getConnectedInstance()));
+    server2.disConnectMember();
```

Comment:
The normal way to shut down a DistributedSystem is to disconnect it.  This method named "disConnectMember" actually crashes the member.  It should be named "crash" or something similar.

Reply:
This disconnect doesn't crash it though, the server is still able to reconnect. Would `forceDisconnect` be better?

Reply:
I think I see what you're saying but a "crashed" node is one that didn't shutdown normally.  The disConnectMember is invoking my MembershipManagerHelper.crashDistributedSystem() method.  That's what I was referring to.  It's not doing a normal disconnect(), so I think it should be renamed in order to avoid confusion.  The crashDistributedSystem method will cause a forced-disconnect.

Reply:
I will rename it to "forceDisconnect"